testSuites/unsecure_testng.xml

```diff
@@ -2,19 +2,19 @@
 <!-- mvn clean test -Dsurefire.suiteXmlFiles=testSuites/unsecure_testng.xml -DSECURE_DRILL=true -->
 <suite name="Unsecure tests" verbose="2" annotations="JDK">
   <listeners>
-    <listener class-name="org.apache.drillui.test.framework.testng.TestEventsListener" />
+    <listener class-name="TestEventsListener" />
   </listeners>
   <test name="a-t1" preserve-order="true" >
     <classes>
-      <class name="org.apache.drillui.test.framework.testng.unsecure.BaseUnsecureTest" />
-      <class name="org.apache.drillui.test.framework.testng.unsecure.nav.NavigationTest" />
-      <class name="org.apache.drillui.test.framework.testng.unsecure.query.DateOutputTest" />
-      <class name="org.apache.drillui.test.framework.testng.unsecure.query.ResultsTableTest" />
-      <class name="org.apache.drillui.test.framework.testng.unsecure.query.ErrorTest" />
-      <class name="org.apache.drillui.test.framework.testng.unsecure.query.ResultsExceptionsTest" />
-      <class name="org.apache.drillui.test.framework.testng.unsecure.query.profiles.ProfileDetailsTest" />
-      <class name="org.apache.drillui.test.framework.testng.unsecure.storage.StoragePageTest" />
-      <class name="org.apache.drillui.test.framework.testng.unsecure.storage.StoragePluginsCRUDTest" />
+      <class name="BaseUnsecureTest" />
```

Comment:
Add packages to the class names, please.

Reply:
Done